dockerfiles/deps/tesseract.sh

```diff
@@ -8,7 +8,7 @@ tar -xzvf leptonica-1.83.1.tar.gz
 cd leptonica-1.83.1 || exit
 ./configure && make && make install
 cd ..
-wget http://mirror.squ.edu.om/gnu/autoconf-archive/autoconf-archive-2017.09.28.tar.xz
+wget https://mirror.team-cymru.com/gnu/autoconf-archive/autoconf-archive-2017.09.28.tar.xz
```

Comment:
The URL was expired, I had to look for a different source. Does this seem legit?

See: https://github.com/Unstructured-IO/base-images/pull/20#issuecomment-2149487254